### DISCUSSION TITLE
Package is not shown in search results

### DISCUSSION THREAD

**Discussion by sinansonmez** [community/community#146157] [npm] [answered]
I published a package 3 days ago. When I search the exact name of the package `tin-validators`, I dont see it in search results

https://www.npmjs.com/package/tin-validators

**Comment by sinansonmez** [comment]
@quoc10973 thanks for your huge effort (!) to pass my question to ChatGPT. I already checked each point in your message

**Reply by leobalter** [reply, marked as answer]
Hi @sinansonmez! No AI response here, directly from the npm team:

It takes a bit of a time for newly published packages to be indexed for search, saying that, we've been noticing our results are aggressively filtering out those recent and/or without much content (on the description and readme). We should have a patch landing soon supporting more packages to be featured on the results.

**Reply by sinansonmez** [reply]
thanks for the answer especially for this `we've been noticing our results are aggressively filtering out those recent and/or without much content (on the description and readme).`. Now it is clear to me
